generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -10,16 +12,31 @@
 from nidaqmx._stubs import nidaqmx_pb2_grpc as nidaqmx_grpc
 from nidaqmx._stubs import session_pb2 as session_grpc_types
 
-
+_logger = logging.getLogger(__name__)
```

Comment:
In addition, we typically do this in the top-level module (e.g. `nidaqmx/__init__.py`): https://github.com/ni/measurementlink-python/blob/main/ni_measurementlink_service/__init__.py#L16

```
_logger = logging.getLogger(__name__)
_logger.addHandler(logging.NullHandler())
```